app/scripts/directives/datepicker.js

```diff
@@ -30,12 +30,13 @@ angular
       },
       link: function (scope, element) {
         var datePickerElement = angular.element(element).find('.datepicker');
-        var initialDate =
+        scope.localModel =
           scope.localModel && scope.monthYearOnly
-            ? scope.localModel
+            ? moment(new Date(scope.localModel)).format('YYYY-MM-DD')
             : scope.localModel
             ? moment(new Date(scope.localModel)).format('MM/DD/YYYY hh:mm A')
             : null;
+        let initialDate = scope.localModel ? scope.localModel : null;
```

Comment:
I'm not the best at concise JS syntax, but would this work?
```
const initialDate = scope.localModal || null;
```

Reply:
yes i believe so